test/e2e/plain_provisioner_test.go

```diff
@@ -247,6 +247,109 @@ var _ = Describe("plain provisioner bundle", func() {
 			}).Should(BeTrue())
 		})
 	})
+
+	When("a bundle containing no manifests is created", func() {
```

Comment:
This seems useful outside of the scope of this PR?

Reply:
yeah +1 i'll make a separate PR for this since we already have code that checks for this.

Reply:
The addition of that skipRootPaths array also seems like it's applicable to the unpacker outside of these changes?

Reply:
In fact, a good chunk of this is applicable without bringing metadata stuff into the equation.

Reply:
Alright broke a good chunk out into #174